packages/api/src/get-test.js

```diff
@@ -78,6 +78,23 @@ class GetTestHandler {
         handleError(res, err);
       }
     });
+
+    // returns all tests that failed or are flaky, and then any remaining tests
+    this.app.get('/api/repo/:orgname/:reponame/tests', async (req, res, next) => {
+      try {
+        const repoid = firebaseEncode(req.params.orgname + '/' + req.params.reponame);
+        const limit = 30;
```

Comment:
I would add a comment that notes we should make result size configurable.

Reply:
Added limit query param